library/data/component/src/main/java/org/quiltmc/qsl/component/api/wrapper/Wrapper.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.component.api.wrapper;
+
+public class Wrapper<T> {
```

Comment:
What's the point of this API class? It only seems to be used in your testmod, and there it can easily be replaced by doing `final T[] tBuf = new T[1];`.